index.html

```diff
@@ -210,7 +210,76 @@ <h4>Skills</h4>
             </ul>
           </section>
         </div>
-    </section>
+      </section>
+      <!--Contact form-->
+      <section class="contact">
+        <h2 class="contact-message">
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </h2>
+        <form
+          action="https://formspree.io/f/mnqrnjnk"
+          method="post"
+          class="contact-form"
+        >
+          <div class="form-element">
+            <input
+              type="text"
+              name="name"
+              id="name"
+              required
+              placeholder=" Full Name"
+              maxlength="30"
+            />
+          </div>
+          <div class="form-element">
+            <input
+              type="email"
+              name="email"
+              id="email"
+              required
+              placeholder=" Email address"
+            />
+          </div>
+          <div class="form-element">
+            <textarea
+              name="message"
+              id="message"
+              required
+              maxlength="500"
+              placeholder=" Enter text here"
+            ></textarea>
+          </div>
+          <div class="form-element btn">
+            <button type="submit" class="sbtn">Get in touch</button>
+          </div>
+        </form>
+      </section>
+      <!--Footer Section-->
+      <footer class="footer">
+        <div class="social-media social-bottom">
+          <a href="https://twitter.com/ndorrh" class="social-links"
+            ><img src="images/icons/twitter.svg" alt="twitter"
+          /></a>
+          <a
+            href="https://www.linkedin.com/in/ndorrh-oswald-ebu-82ab02236/"
+            class="social-links"
+            ><img src="images/icons/Linkedin.svg" alt="linkedin"
+          /></a>
+          <a href="#" class="social-links"
+            ><img src="images/icons/medium.svg" alt="medium"
+          /></a>
+          <a href="https://github.com/ndorrh" class="social-links"
+            ><img src="images/icons/github.svg" alt="github"
+          /></a>
+          <a href="#" class="social-links"
+            ><img src="images/icons/angellist.svg" alt="angellist"
+          /></a>
+        </div>
+        <div>
+          <address class="address-style">myemailhere@email.com</address>
+        </div>
+      </footer>
     </main>
   </body>
 </html>
```

Comment:
- Kindly make your page fix, not move from side to side it will remove the scrolling button on the bottom of your page and fix some size and make your design matches more to the Figma. You must stick to the Figma  design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A988) :grinning: :grinning: :grinning: 
 
## your design :grinning: :grinning: 
    
![image](https://user-images.githubusercontent.com/95297251/186281086-02540d64-2607-4219-a40e-3c8ebc125510.png)


## Figma design :+1:  :+1:  :+1: 

![image](https://user-images.githubusercontent.com/95297251/186278974-cd6e0340-4ea0-4c26-83ee-6e49f5f0b03c.png)